code/game/jobs/faction/faction.dm

```diff
@@ -69,12 +69,12 @@
 	switch(mission_level)
 		if(REPRESENTATIVE_MISSION_HIGH)
 			objective = pick("Assist your contractor in smuggling [rand(1,4)] items of value",
-							"Collect evidence of Nanotrasen being unfair or oppressive against your contractors to be used as leverage in future talks")
+							"Collect evidence of the SCC being unfair or oppressive against your contractors to be used as leverage in future talks")
```

Comment:
Can use `current_map.company_short` here and elsewhere.

Reply:
replaced where I could, but I didn't really want to override New() 100 times just to make it dynamic for this specifically